<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-I1E13DA" failures="4" tests="9" name="Surefire test" time="18.973" errors="0" timestamp="2023-11-29T21:53:53 IST">
  <testcase classname="api.test.DataDrivenTest" name="testPostUsers" time="5.018"/>
  <testcase classname="api.test.DataDrivenTest" name="testPostUsers" time="1.128"/>
  <testcase classname="api.test.DataDrivenTest" name="testPostUsers" time="0.001">
    <failure type="java.lang.NumberFormatException" message="Cannot parse null string">
      <![CDATA[java.lang.NumberFormatException: Cannot parse null string
at java.base/java.lang.Integer.parseInt(Integer.java:630)
at java.base/java.lang.Integer.parseInt(Integer.java:786)
at api.test.DataDrivenTest.testPostUsers(DataDrivenTest.java:20)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 32 stack frames]]>
    </failure>
  </testcase> <!-- testPostUsers -->
  <testcase classname="api.test.DataDrivenTest" name="testDeleteUserByName" time="1.194"/>
  <testcase classname="api.test.DataDrivenTest" name="testDeleteUserByName" time="1.14"/>
  <testcase classname="api.test.DataDrivenTest" name="testDeleteUserByName" time="1.138">
    <failure type="java.lang.AssertionError" message="expected [200] but found [404]">
      <![CDATA[java.lang.AssertionError: expected [200] but found [404]
at api.test.DataDrivenTest.testDeleteUserByName(DataDrivenTest.java:35)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 38 stack frames]]>
    </failure>
  </testcase> <!-- testDeleteUserByName -->
  <testcase classname="api.test.UserTests" name="testPostUser" time="1.338"/>
  <testcase classname="api.test.UserTests" name="testGetUserByName" time="1.136">
    <failure type="java.lang.AssertionError" message="expected [200] but found [405]">
      <![CDATA[java.lang.AssertionError: expected [200] but found [405]
at api.test.UserTests.testGetUserByName(UserTests.java:47)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 38 stack frames]]>
    </failure>
  </testcase> <!-- testGetUserByName -->
  <testcase classname="api.test.UserTests" name="testUpdateUserByName" time="2.247">
    <failure type="java.lang.AssertionError" message="expected [200] but found [405]">
      <![CDATA[java.lang.AssertionError: expected [200] but found [405]
at api.test.UserTests.testUpdateUserByName(UserTests.java:64)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 38 stack frames]]>
    </failure>
  </testcase> <!-- testUpdateUserByName -->
</testsuite> <!-- Surefire test -->
